tests/unittest/_torch/cute_dsl/test_kimi_k3_kda_ptx_patch.py

```
@@ -0,0 +1,168 @@
+"""
+
+
+def _run_import(inject: str = "") -> subprocess.CompletedProcess:
+    code = _FAKE_DSL.format(fake_body=inject) if inject else ""
+    code += _LOAD_K4.format(k4=str(_k4_path()))
+    return subprocess.run([sys.executable, "-c", code], capture_output=True, text=True, timeout=600)
+
+
+@needs_kernel_deps
+def test_import_applies_patch():
+    proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)